### PR TITLE
OCPBUGS-31088: Fix loopvar issue

### PR DESCRIPTION
**Pull request by fxierh**
Fix loopvar issue in a range loop.

### CODE REVIEW

**Comment by fxierh** [comment]
@jparrill @bryan-cox @muraee @Patryk-Stefanski Could you please take a look ? Thanks.

File: cmd/infra/azure/create.go [hunk 1]
```diff
@@ -270,9 +270,9 @@ func createResourceGroup(ctx context.Context, o *CreateInfraOptions, azureCreds
 	} else {
 
 		resourceGroupTags := map[string]*string{}
-
 		for key, value := range o.ResourceGroupTags {
-			resourceGroupTags[key] = &value
+			valueCopy := value
+			resourceGroupTags[key] = &valueCopy
```
Comment (muraee) [inline cmd/infra/azure/create.go:275]:
```suggestion
			resourceGroupTags[key] = ptr.To(value)
```

Comment (fxierh) [reply cmd/infra/azure/create.go]: Updated.

**Comment by fxierh** [comment]
@Patryk-Stefanski Done. Let's see if the Konflux thing will pass this time.

**Comment by fxierh** [comment]
Okay `Red Hat Konflux / hypershift-operator-hhlfs / hypershift-main-enterprise-contract` failed again. It just won't work..

**Comment by bryan-cox** [comment]
/override "Red Hat Konflux / hypershift-operator-wfjcb / hypershift-main-enterprise-contract"

**Pull request merged**